### PR TITLE
refactor: use `registerKeyset`

### PR DESCRIPTION
**Pull request by ogonkov**

### CODE REVIEW

File: src/components/Breadcrumbs/i18n/index.ts [hunk 1]
```diff
@@ -1,11 +1,7 @@
-import {i18n} from '../../../i18n';
-import {Lang} from '../../utils/configure';
+import {registerKeyset} from '../../utils/registerKeyset';
 import en from './en.json';
 import ru from './ru.json';
 
 const COMPONENT = 'Breadcrumbs';
```
Comment (ValeraS) [inline src/components/Breadcrumbs/i18n/index.ts:5]: Let's use for ids a name like this `yc-some-component-name`.

Comment (ogonkov) [reply src/components/Breadcrumbs/i18n/index.ts:5]: Can you propose changes please?

Comment (ogonkov) [reply src/components/Breadcrumbs/i18n/index.ts:5]: Isn't it a breaking change if someone override it in project?

Comment (ValeraS) [reply src/components/Breadcrumbs/i18n/index.ts:5]: `i18n` will throw error if you try to use the same name for a keyset https://github.com/yandex-cloud/i18n/blob/4f791c56822dc9d1645cbe852738056412d5d0e6/src/index.ts#L26

**Review by vrozaev — APPROVED** [review]

**Review by ValeraS — APPROVED** [review]

**Review by amje — APPROVED** [review]

**Pull request merged**